.gitignore

```diff
@@ -133,3 +133,7 @@ dmypy.json
 
 # Pyre type checker
 .pyre/
+
+.DS_Store
+.DS_Store
+.DS_Store
```

Comment:
It doesnt need to be changed but I wanted to let you know I thought this was very funny

Reply:
occult rituals to banish .DS_store